src/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { Person } from './types';
+
+interface PersonLinkProps {
+  person: Person;
+  isMother?: boolean;
+}
+
+export const PersonLink: FC<PersonLinkProps> = ({ person, isMother }) => (
+  <>
+    {person.name ? (
```

Comment:
why do we need this condition?
if i understand correctly, this component can only get person as an object, because you've checked it in the `createPerson` function